core/root/operation_targets.go

```diff
@@ -4,27 +4,18 @@ import "strings"
 
 // TODO: Add etcd
```

Comment:
Don't know if this TODO comment is still relevant?

Reply:
Seems irrelevant now. Good catch!